### ISSUE TITLE
created a virus

### ISSUE THREAD

**Issue by aspaceghost** [Dhruv-Techapps/schedule-url#2]
When I selected the DOCS link my screen was overwhelmed by garbage

**Comment by dharmesh-hemaram** [comment]
really

**Comment by aspaceghost** [comment]
Yes. It took over the screen with some kind of warning or threat … don’t 
remember exactly what. Did not lock up the computer though


------ Original Message ------
From "Dharmesh" ***@***.***>
To "Dhruv-Techapps/schedule-url" ***@***.***>
Cc "aspaceghost" ***@***.***>; "Author" 
***@***.***>
Date 10/4/2024 11:43:58 PM
Subject [CATCHALL] Re: [Dhruv-Techapps/schedule-url] created a virus 
(Issue #2)
